src/main/resources/application.properties

```diff
@@ -1,11 +1,10 @@
 spring.application.name=Service
 
 # MySQL connection configuration
-# Current IP address of Cloud SQL instance: 34.73.24.125
-spring.datasource.url=jdbc:mysql://34.73.24.125:3306/reservation_management
-#spring.datasource.url=jdbc:mysql://localhost:3306/reservation_management
-spring.datasource.username=root
-spring.datasource.password=dbuserdbuser
+# URL for MySQL DB for local testing
+spring.datasource.url=${DB_URL}
```

Comment:
This is really good. If we have time lets find a way to hook these up to some config local files so we dont have to constantly feed in credentials to connect to the service